examples/redis_sync/.fmf/version

```diff
@@ -0,0 +1 @@
+1
```

Comment:
Would you be ok with naming the directory just `redis`? Or use dash to be consistent with most of the file names.

Reply:
sure, no problem, just wanted to mention that it is sync, in case someone will go to examples directly.